content/docs/start/metrics-parameters-plots.md

```diff
@@ -0,0 +1,257 @@
+[was created](/doc/start/data-pipelines#dependency-graphs-dags) with this
+`dvc run` command. Notice the argument sent to the `-p` option (short for
+`--params`):
+
+```dvc
+$ dvc run -n featurize \
+          -p featurize.max_features,featurize.ngrams \
+          -d src/featurization.py -d data/prepared \
+          -o data/features \
+          python src/featurization.py data/prepared data/features
+```
+
+</details>
+
+The `params` section defines the [parameter](/doc/command-reference/params)
+dependencies of the `featurize` stage. By default DVC reads those values
+(`featurize.max_features` and `featurize.ngrams`) from a `params.yaml` file. But
+as with metrics and plots, parameter file names and structure can also be user
+and case defined.
+
+This is how our `params.yaml` file looks like:
+
+```yaml
+prepare:
+  split: 0.20
+  seed: 20170428
+
+featurize:
+  max_features: 500
+  ngrams: 1
+
+train:
+  seed: 20170428
+  n_est: 50
+  min_split: 2
+```
+
+## Updating and running
+
+Let's edit the `params.yaml` file to use bigrams and increase the number of
+features:
+
+```diff
+ featurize:
+-  max_features: 500
+-  ngrams: 1
++  max_features: 1500
++  ngrams: 2
+```
+
+And the beauty of `dvc.yaml` is that all you need to do now is to run:
+
+```dvc
+$ dvc repro
+```
+
+It'll analyze the changes, use existing cache of previous runs, and execute only
+the commands that are needed to get the new results (model, metrics, plots).
+
+The same logic applies to other possible adjustments — edit source code, update
+datasets — you do the changes, use `dvc repro`, and DVC runs what needs to be
+run.
+
+## Comparing changes
+
+Finally, let's see how the updates improved performance. DVC has a few commands
+to see metrics and parameter changes, and to visualize plots, for one or more
+pipeline iterations. Let's compare the current "bigrams" run with the last
+committed "baseline" iteration:
+
+```dvc
+$ dvc params diff
+Path         Param                   Old    New
+params.yaml  featurize.max_features  500    1500
+params.yaml  featurize.ngrams        1      2
+```
+
+`dvc params diff` can show how params in the workspace differ vs. the last
+commit.
+
+`dvc metrics diff` does the same for metrics:
+
+```dvc
+$ dvc metrics diff
+Path         Metric    Old      New      Change
+scores.json  avg_prec  0.52048  0.55259  0.03211
+scores.json  roc_auc   0.9032   0.91536  0.01216
+```
+
+Since there are multiple arrays of data in each plots file, let's specify which
+to use as the axes in our plots:
+
+```dvc
+$ dvc plots modify prc.json -x recall -y precision
+Modifying stage 'evaluate' in 'dvc.yaml'
+$ dvc plots modify roc.json -x fpr -y tpr
+Modifying stage 'evaluate' in 'dvc.yaml'
+```
+
+And finally, we can compare both `precision recall` and `roc` curves with a
+single command!
+
+```dvc
+$ dvc plots diff
+file:///Users/dvc/example-get-started/plots.html
+```
+
+![](/img/plots_prc_get_started.svg) ![](/img/plots_roc_get_started.svg)
```

Comment:
Also I just noticed `plots` are only shown all the way down here at compare (`plots diff`). Out of scope prob, but since there are some other non-experiment-related changes to this doc, we could throw in a `plots show` example earlier in the page.

Reply:
In the top "Collecting metrics" section, the internals of the metrics and plots files are shown but `metrics show` and `plots show` are not used. Adding those commands might be useful, and maybe the file details should even be moved into a collapsible?

Reply:
Since this is out of scope, I'm going to put it in a separate PR.

Reply:
Added PR #2240

Reply:
Thanks, yeah this was OOS